### PR TITLE
avocado/utils/kernel.py: fix the use of the asset library

### PR DESCRIPTION
**Pull request by clebergnu**
And the docstring of the asset library itself.

Signed-off-by: Cleber Rosa <crosa@redhat.com>

### CODE REVIEW

File: avocado/utils/kernel.py [hunk 1]
```diff
@@ -72,7 +72,8 @@ def download(self):
         full_url = self.URL + self.SOURCE.format(version=self.version)
         self.asset_path = asset.Asset(full_url, asset_hash=None,
                                       algorithm=None, locations=None,
-                                      cache_dirs=self.data_dirs).fetch()
+                                      cache_dirs=[self.work_dir], expire=0).fetch()
```
Comment (apahim) [inline avocado/utils/kernel.py]: Always expire? Why not use the cached version?

File: avocado/utils/asset.py [hunk 2]
```diff
@@ -50,8 +50,8 @@ def __init__(self, name, asset_hash, algorithm, locations, cache_dirs,
         :param asset_hash: asset hash
         :param algorithm: hash algorithm
         :param locations: list of locations fetch asset from
-        :params cache_dirs: list of cache directories
-        :params expire: time in seconds for the asset to expire
+        :param cache_dirs: list of cache directories
+        :param expire: time in seconds for the asset to expire
```
Comment (apahim) [inline avocado/utils/asset.py]: Good catch.

File: avocado/utils/kernel.py (hunk 1 above)
Comment (clebergnu) [inline avocado/utils/kernel.py]: The point is that `work_dir` (as a cache_dir) is going to be a new one anytime, anyway.  Actually, I was wondering if it's possible to have a "library default" cache dir, that could be used as a default value for `cache_dirs`.

Comment (apahim) [inline avocado/utils/kernel.py]: Ok, got it. I think we should keep `expire=None` here anyway, so we don't have to revisit when we have a persistent cache dir.

Comment (clebergnu) [inline avocado/utils/kernel.py]: NP, I have no strong opinions on that (other than the "library default cache dir" idea).

**Pull request closed without merging**